imsim/readout.py

```diff
@@ -206,15 +206,14 @@ def cte_matrix(npix, cti, ntransfers=20):
     return my_matrix
 
 
-# TODO: get lsst_num from camera object.
-def get_primary_hdu(eimage, lsst_num='LCA-11021_RTM-000', camera_name=None,
+def get_primary_hdu(eimage, lsst_num, camera_name=None,
```

Comment:
Is lsst_num really the right name for this parameter?  It doesn't look like a number.  Maybe `lsst_serial` is more apt, since the value comes from `ccd.getSerial()`.

Reply:
The project header keyword name for this value is `LSST_NUM`, hence the name used here.   We can use `lsst_serial` if you want, but that's the reason `lsst_num` was originally selected.